akka-stream/src/main/resources/reference.conf

```diff
@@ -47,6 +47,9 @@ akka {
     # such as FileSource, FileSink and others.
     file-io-dispatcher = "akka.stream.default-file-io-dispatcher"
 
+    # Blocking IO like OutputStreamSource and InputStreamSink uses by default the same settings
+    blocking-io-dispatcher = "akka.stream.default-file-io-dispatcher"
```

Comment:
Ok, just in case. I'll rename streams file-io-dispatcher. 
akka-actor will stay with it's
```file-io-dispatcher = "akka.actor.default-dispatcher"``` and akka-http will point to blocking ```file-io-dispatcher = ${akka.stream.blocking-io-dispatcher}```